packages/dcos-integration-test/test_signal.py

```diff
@@ -0,0 +1,132 @@
+"""
+Test Enterprise DC/OS Signal Service
+TODO: this test only differs from upstream in the services that it checks for, rather
+    find a method such that we do not need to duplicate so much code
+"""
+import json
+import logging
+import os
+import subprocess
+from pathlib import Path
+
+from dcos_test_utils.enterprise import EnterpriseApiSession
+from ee_helpers import get_dcos_config
+
+__maintainer__ = 'mnaboka'
```

Comment:
:fire:

Reply:
To my defense I'd like to say that I just copied that complete file from mesosphere/dcos-enterprise.